R/wrappers.R

```diff
@@ -16,3 +16,5 @@ isRealReallyInt = function(x) .Call(CisRealReallyIntR, x)
 isReallyReal = function(x) .Call(CisReallyReal, x)
 
 coerceAs = function(x, as, copy=TRUE) .Call(CcoerceAs, x, as, copy)
+
+frev = function(x, copy=TRUE) if (isTRUE(copy)) .Call(Cfrev, x, copy) else invisible(.Call(Cfrev, x, copy))
```

Comment:
WDYT about a new function in utils `visible = identity` then

`frev = function(x, copy=TRUE) (if (isTRUE(copy)) visible else invisible)(.Call(Cfrev,x, copy))`

Or we can just use `identity()`? Current code is a bit hard to read since my instinct is to scan letter-by-letter for what's the difference between the `if`/`else` branches.

Reply:
using `(expr eval to fun)(fun args)` is less frequently used to may be less readable at first

Reply:
done